### DISCUSSION TITLE
Support for GRPC

### DISCUSSION THREAD

**Discussion by shalousun** [TongchengOpenSource/smart-doc#553] [Ideas]
Support Generate API for GRPC，Community developers are welcome to join in the co-development.
smart-doc需要新增GRPC协议的文档生成支持，欢迎社区用户参与共建

**Comment by Alicent1024** [comment]
Hello, could you please let me know how this task is progressing?
你好，请问这个目前进度如何？

**Reply by shalousun** [reply]
@Alicent1024 The current community hasn't had time to work on it yet. If you are interested, you are welcome to participate in the joint construction.
当前社区还没时间去开发，如果你感兴趣，欢迎参与共建

**Reply by linwumingshi** [reply]
> 你好，请问这个目前进度如何？

#851

**Reply by shalousun** [reply]
The PR supporting gRPC has been merged, and the community will release this feature in version 3.0.7.
